config/settings.yml

```diff
@@ -44,6 +44,12 @@ goobi:
 
 gis:
   robots_content_root: '/gis_workflow_data/stage'
-  username: 'lyberadmin'
 
```

Comment:
do not need this anymore, the GIS test data can now be found on the pre-assembly server because of the new mount, so one less server to SSH into